src/test/java/recipeditor/command/FindCommandTest.java

```diff
@@ -1,33 +1,42 @@
 package recipeditor.command;
 
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 import recipeditor.parser.Parser;
 import recipeditor.recipe.Ingredient;
 import recipeditor.recipe.Recipe;
 import recipeditor.recipe.RecipeList;
 import recipeditor.storage.Storage;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 public class FindCommandTest {
 
+    private static Recipe recipe;
+
+    @BeforeAll
+    static void setUp() {
+        recipe = new Recipe("Example title", "Example description");
```

Comment:
I used `test title`(which I still feel is not a very safe file name), `Example Title` can easily collide with user input recipes, and disrupt the saved files. Maybe use some names that users cannot possibly type in, like `v0&9+Qg"<`

Reply:
Thanks for pointing that out! I will do my testing on them and check if it affects user's previously saved files.